inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,61 @@
+package inheritance;
+
+import java.util.List;
+
+public class Restaurant {
+    private String name;
+    private int stars;
+    private String priceCategory;
+    //get bunch of reviews for the restaurant
+    public List<Review> reviews;
+
+//setters
+    public void setName(String name) {
+        this.name = name;
+    }
+
+
+    public void setPriceCategory(String priceCategory) {
+        this.priceCategory = priceCategory;
+    }
+
+//getters
+    public String getName(){
+        return this.name;
+    }
+    public  int getStars(){
+        return this.stars;
+    }
+    public String getPriceCategory(){
+        return this.priceCategory;
+    }
+
+    //restaurant constructor
+    public Restaurant(String name, String priceCategory){
+        this.name = name;
+        this.stars = 0;
+        this.priceCategory = priceCategory;
```

Comment:
You could have `this.reviews = new ArrayList<Review>` here so that each new instance of the Restaurant class would immediately have a list to add reviews to